proto/cmp/services/insurance/v2/insurance_types.proto

```diff
@@ -0,0 +1,171 @@
+syntax = "proto3";
+
+package cmp.services.insurance.v2;
+
+import "cmp/types/v1/inclusivity.proto";
+import "cmp/types/v1/link.proto";
+import "cmp/types/v2/contact_info.proto";
+import "cmp/types/v2/price.proto";
+import "cmp/types/v2/product_code.proto";
+import "cmp/types/v3/file.proto";
+import "google/protobuf/timestamp.proto";
+
+// Insurance
+//
+// ![Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/insurance/v2/insurance_types.proto.dot.xs.svg)
+//
+// [Open Message Diagram](https://storage.googleapis.com/docs-cmp-files/diagrams/proto/cmp/services/insurance/v2/insurance_types.proto.dot.svg)
+message Insurance {
+  // Context for Inventory system concepts that need to be included in an info
+  // response, like an OwnerCode, PTC_OfferParameters, Tax codes, Disclosure RefID,
+  // etc. or a serialized combination of these codes.
+  string context = 1;
+
+  // An identifier for external sessions, aiding in tracking and continuity across
+  // sessions.
+  string external_session_id = 2;
+
+  // List of insurance policies
+  repeated cmp.services.insurance.v2.Policy policies = 3;
+}
+
+message Policy {
+  // Product Code
+  cmp.types.v2.ProductCode product_code = 1;
+
+  // Description
+  string description = 2;
+
+  // Ex: "2023-08-28T12:03:50", specifying when the static data of a product was
+  // last updated
+  //
+  // Timestamps may be used for both off-chain and on-chain operations.
+  // For on-chain operations, only seconds are supported, and nanoseconds
+  // will be ignored.
+  google.protobuf.Timestamp last_modified = 3;
+}
+
+message PolicyExtendedInfo {
+  // Insurance Policy dynamic info
```

Comment:
```suggestion
  // Insurance Policy static info. These details are only pulled after the InsuranceProductList was pulled and the Timestamp in the response indicated that the policy was updated since the last time the list was pulled.
```